scripts/travis/06-test-source-code.sh

```diff
@@ -10,6 +10,12 @@ phpcs --standard=DrupalPractice --extensions=php,module,inc,install,test,profile
 if ! [ -x "$(command -v eslint)" ]; then
     npm install -g eslint
 fi
+
+# Install ESLint requirements
+npm install -g eslint-config-airbnb eslint-plugin-jsx-a11y eslint-plugin-react eslint-plugin-import
+npm install -g prettier eslint-config-prettier eslint-plugin-prettier
```

Comment:
https://www.drupal.org/node/1955232 says we can just install  eslint-config-drupal  to get all required dependencies